stats_scripts/create_graphs.py

```diff
@@ -156,12 +181,12 @@ def mkdir(dir):
         for index, name in enumerate(parameter_names): #Set config
             locals()[name] = treatment[index]
         SIDE = int(sqrt(POP))
-        SEED = (HRR,SLR,BS,BT,SL,SIDE).__hash__()
+        SEED = 0#(HRR,SLR,BS,BT,SL,SIDE).__hash__()
```

Comment:
This is a welcome change, but missed by the commit message. This makes all treatments have the same list of random seeds.